### ISSUE TITLE
Instance fields accessible in static method

### ISSUE THREAD

**Issue by disarray2077** [beefytech/Beef#1432]
Somehow the code below currently compiles:

```
using System;

namespace test6
{
	struct STest
	{
	}

	class Program
	{
		STest test;

		public static int Main(String[] args)
		{
			test = .();

			return 0;
		}
	}
}
```

Tested with https://github.com/beefytech/Beef/commit/e7fe91facb9e0c535be652594bb46cdeffb23e91

**Comment by bfiete** [comment]
Well, it's all an illusion anyway since `test` is valueless. But, yeah, still should have been an error.

**Comment by bfiete** [comment]
Fixed at b0c6dd7e4366596059ba70ae3e27e0fbbdfad21d
